pkg/server/server.go

```
@@ -2,91 +2,93 @@ package server
+	return &Server{
+		cfg:  cfg,
+		app:  setupFiberApp(cfg),
+		deps: deps{},
+	}
+}
 
-	routerService := router.NewRouter(app, cfg)
-	routerService.RegisterRoutes()
+func (s *Server) Start(ctx context.Context) error {
+	s.deps.binanceWS = websocket.NewBinanceWebSocket(ctx, s.cfg)
+	s.deps.binanceWS.Start()
```

golangci-lint (1.62.2)

Error return value of `s.deps.binanceWS.Start` is not checked

(errcheck)